Sources/SwiftParser/Expressions.swift

```diff
@@ -1875,6 +1879,9 @@ extension Parser {
     var keepGoing: RawTokenSyntax? = nil
     var loopProgress = LoopProgressCondition()
     repeat {
+      if allowTrailingComma, currentToken.rawTokenKind == .rightParen {
```

Comment:
Similar comment as in other places, I would prefer this to be centralized in the `repeat while` condition.

Also

```suggestion
      if allowTrailingComma, self.at(.rightParen) {
```